recipes/dirent-path-to-parent-path/README.md

```diff
@@ -0,0 +1,51 @@
+# `dirent.path` DEP0178
+
+This codemod transforms the usage of `dirent.path` to use `dirent.parentPath`.
+
+See [DEP0178](https://nodejs.org/api/deprecations.html#DEP0178).
+
+## Example
+
+**Before:**
+
+```js
+const { readdir } = require('node:fs/promises');
+
+const entries = await readdir('/some/path', { withFileTypes: true });
+for (const dirent of entries) {
+  console.log(dirent.path);
+}
+```
+
+**After:**
+
+```js
+const { readdir } = require('node:fs/promises');
+
+const entries = await readdir('/some/path', { withFileTypes: true });
+for (const dirent of entries) {
+  console.log(dirent.parentPath);
+}
+```
```

Comment:
```suggestion
```diff
  const { readdir } = require('node:fs/promises');

  const entries = await readdir('/some/path', { withFileTypes: true });
  for (const dirent of entries) {
-   console.log(dirent.path);
+   console.log(dirent.parentPath);
  }
```
```